idd/Energy+.idd.in

```diff
@@ -16508,57 +16519,50 @@ SurfaceProperty:ExposedFoundationPerimeter,
        \minimum 0.0
        \maximum 1.0
        \default 1.0
```

Comment:
I can't see the context here. Are there changes needed to the two fields shown below?

```
  N1 , \field Total Exposed Perimeter
       \type real
       \units m
       \minimum> 0.0
  N2 , \field Exposed Perimeter Fraction
       \type real
       \units dimensionless
       \minimum 0.0
       \maximum 1.0
```